build/yamls/base/role.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Just want to know how this file is generated, copy from somewhere?

Reply:
@jwsui I've tried to gather the core roles which are required by the operator itself (vs the ones which are required for granting to Antrea components).
I'm not sure but maybe it's the same purpose as config/rbac/role.yaml - which should be revised in any case as it still contains references to antrea.tanzu.vmware.com which are deprecated.
So maybe I could revise config/rbac/role.yaml and use it instead - and discard build/yamls/base/role.yaml. Do you have an opinion or feedback about this?

Reply:
Yes, you are right that config/rbac/role.yaml should be revised as it's too old! And I also agree to revise config/rbac/role.yaml and use it instead as it has the same purpose as build/yamls/base/role.yaml.

In addition, when revising config/rbac/role.yaml, I'd suggest to start with updating the 'kubebuilder label' with core operator roles[labels](https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/controllers/antreainstall_controller.go#L337), and generate a new config/rbac/role.yaml just by `make manifests`.